test/extension.test.js

```diff
@@ -164,8 +164,32 @@ suite('main', function () {
 
     vsCodeMock.window.activeTextEditor.selection.active.line = 5;
 
-    let url = await main.getGithubUrl(vsCodeMock);
+    let url = await main.getGithubUrl(vsCodeMock, { platform: 'win32' });
```

Comment:
One more `platform` config to remove.

Reply:
Sorry for that! I removed a comment about config too. Thanks!